### PR TITLE
apt: add page

### PR DESCRIPTION
**Pull request by fuerbringer**
Since #1164 was closed I thought I might give it a try. I included the `purge` command in the description of `remove` because the page already has more than the recommended amount of commands (5-6).
Also included the `search` command, since @agnivade mentioned it in #1164. Most of the other commands are adapted or taken directly from the `apt-get` page.

Thanks for reviewing

### CODE REVIEW

File: pages/linux/apt.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# apt
+
+> Package management utility for Debian based distributions. 
+
+- Synchronize list of packages and versions available. This should be run first, before running subsequent apt commands:
```
Comment (waldyrious) [inline pages/linux/apt.md]: "first, " can be removed to shorten the sentence. Also "synchronize" --> "update" and "subsequent" --> "further" (mostly to shorten the sentence some more).

File: pages/linux/apt.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- Search for packages:
+
+`apt search {{package}}`
+
+- Install a new package:
+
+`apt install {{package}}`
+
+- Remove a package (using "purge" instead also removes configuration files):
```
Comment (waldyrious) [inline pages/linux/apt.md]: Suggestion: "...also removes *its* configuration files)"

File: pages/linux/apt.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+`apt remove {{package}}`
+
+- Upgrade installed packages to newest available versions:
```
Comment (waldyrious) [inline pages/linux/apt.md]: "to *the* newest"

File: pages/linux/apt.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+`apt upgrade`
+
+- Remove no longer needed packages:
+
+`apt autoremove`
+
+- Upgrade installed packages (like "upgrade"), but remove obsolete packages and install additional packages to meet new dependencies:
```
Comment (waldyrious) [inline pages/linux/apt.md]: Does the "remove obsolete packages" match the behavior of `apt autoremove`? If so, the descriptions should use the same wording. Also, is "install additional packages to meet new dependencies" not true with a plain upgrade?

**Comment by waldyrious** [comment]
Nice one @fuerbringer, thanks! Please check the comments I left inline. Also, @agnivade any clue why tldr-bot didn't manifest here?

**Comment by waldyrious** [comment]
Looks great @fuerbringer, thanks :)

**Comment by agnivade** [comment]
@waldyrious - Hmm .. strange, travis didn't set any environment variables at all. So the python script failed.  Compare https://travis-ci.org/tldr-pages/tldr/builds/184201878 and https://travis-ci.org/tldr-pages/tldr/builds/185003321. See that in the first case, travis sets env vars from repository settings and from .travis.yml, in the 2nd it does none.

Is there any condition where travis does not export env vars ?

**Comment by waldyrious** [comment]
> Is there any condition where travis does not export env vars ?

I have no idea, I know very little of Travis setup :/

**Comment by fuerbringer** [comment]
Running `tldr-lint` locally returns this: `pages/linux/apt.md:3: TLDR004 Command descriptions should end in a period`. How do fix this?

**Comment by fuerbringer** [comment]
The error was caused by a space after the period in the command description. Removing it fixed the Travis tests.

**Comment by waldyrious** [comment]
Nice find! @agnivade we definitely should fix the linter so it gives the correct hint here. Would you like to take a shot at that?

**Comment by agnivade** [comment]
@waldyrious - Sure.

**Comment by waldyrious** [comment]
As for this PR, I think it's ready to merge. LGTM from me.

**Comment by waldyrious** [comment]
Thanks for the contribution and the patience, @fuerbringer.

**Pull request merged**